pages/common/csslint.md

```diff
@@ -0,0 +1,27 @@
+# csslint
+
+> A linter for CSS code.
+
+- Lint a single CSS file:
+
+`csslint {{file.css}}`
+
+- Lint multiple CSS files:
+
+`csslint {{file1.css}} {{file2.css}} {{file3.css}}`
+
+- List all possible style rules:
+
+`csslint --list-rules`
+
+- Specify certain rules as errors (which result in a non-zero exit code):
+
+`csslint --errors={{rule1}},{{rule2}},{{rule3}} {{file.css}}`
```

Comment:
I would prefer to mention some concrete rules here. Something like `csslint --errors={{floats,important}} {{file.css}}`. And same for the next 2 examples.